starters/standard/wrangler.jsonc

```diff
@@ -41,7 +41,7 @@
   "migrations": [
     {
       "tag": "v1",
-      "new_classes": ["SessionDurableObject"]
+      "new_sqlite_classes": ["SessionDurableObject"]
```

Comment:
From https://blog.cloudflare.com/sqlite-in-durable-objects/:

> SQLite-backed objects also support the existing [key/value-based storage API](https://developers.cloudflare.com/durable-objects/api/transactional-storage-api/): KV data is stored into a hidden table in the SQLite database. So, existing applications built on DOs will work when deployed using SQLite-backed objects.

From what I understand, the KV storage API is still supported - I think this makes sense - sometimes you don't want the structure, and just want keys and values.

In contrast, the KV _backend_ is superseded by the sqlite backend. This changes us to using the sqlite backend. So it seems its all we need to do, and the we can continue to use the KV API.